### DISCUSSION TITLE
release/note

### DISCUSSION THREAD

**Discussion by giscus** [gogf/gf-site#85] [General]
# release/note

GoFrame框架的版本发布说明，包括使用GNU风格的版本命名规则，如何保证版本间的兼容性，以及推荐的版本升级方式。此外，文中还介绍了master和develop分支的使用注意事项及操作方法，以确保开发者能够高效地管理和升级框架版本。

https://goframe.org/release/note

**Comment by Abner3721** [comment]
学习
